### PR TITLE
Added Continuous Integration with TravisCI

### PR DESCRIPTION
**Pull request by Penchant**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - oraclejdk8
+env:
+  global:
+    - JAVA_HOME=/usr/lib/jvm/java-8-oracle
+before_install:
+    - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+   # - export '%JAVA_HOME%'=$(JAVA_HOME)
+script:
+    - echo $JAVA_HOME
+#    - cp -r src out
```
Comment (DanielChurch) [inline .travis.yml]: Remove the commented out code

File: build.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+path.variable.kotlin_bundled=C\:\\Program Files\\JetBrains\\IntelliJ IDEA Community Edition 2017.2.4\\plugins\\Kotlin\\kotlinc
+path.variable.maven_repository=C\:\\Users\\Dylan\\.m2\\repository
```
Comment (DanielChurch) [inline build.properties]: Should this file be included?

**Review by DanielChurch — CHANGES_REQUESTED** [review]
Looks good, builds good, but a few changes :)

**Comment by Penchant** [comment]
Requested changes have been implemented

**Review by DanielChurch — APPROVED** [review]
+1 Looks pretty clean to me :)

**Pull request merged**